server/src/main/java/org/apache/iotdb/db/engine/snapshot/SnapshotTaker.java

```diff
@@ -183,8 +186,8 @@ private void createHardLink(File target, File source) throws IOException {
       LOGGER.error("Hard link source file {} doesn't exist", source);
     }
     Files.deleteIfExists(target.toPath());
-    Files.createLink(target.getAbsoluteFile().toPath(), source.getAbsoluteFile().toPath());
-    snapshotLogger.logFile(source.getAbsolutePath(), target.getAbsolutePath());
```

Comment:
Is this change equivalent with previous style ？

Reply:
yes